src/masks/InteractionMasks.tsx

```diff
@@ -172,66 +120,96 @@ export default class InteractionMasks<R, K extends keyof R> extends React.Compon
       left: selectionMaskLeft - portalTargetLeft + scrollLeft,
       top: selectionMaskTop - portalTargetTop + scrollTop
     };
+  }, [editorPortalTarget]);
+
+  // Focus on the selection mask when the selected position is changed
+  useEffect(() => {
+    if (selectedPosition.rowIdx === -1 || selectedPosition.idx === -1) return;
+    selectionMaskRef.current?.focus();
+  }, [selectedPosition]);
+
+  // Focus on the selection mask after the editor is closed
+  useEffect(() => {
+    if (isEditorEnabled) return;
+    selectionMaskRef.current?.focus();
+  }, [isEditorEnabled]);
+
+  useEffect(() => {
+    if (!enableEditorAfterRender || !isCellEditable) return;
+    setEnableEditorAfterRender(false);
```

Comment:
Shouldn't we call this whether the cell is editable or not?

Reply:
Fixed